apmpackage/apm/manifest.yml

```diff
@@ -142,7 +142,7 @@ policy_templates:
           - name: tls_supported_protocols
             type: text
             multi: true
-            default: ["TLSv1.0", "TLSv1.1", "TLSv1.2"]
+            default: ["TLSv1.0", "TLSv1.1", "TLSv1.2", "TLSv1.3"]
```

Comment:
I am fairly certain that `"TLSv1.0"` should not be a default option any more but was re-introduced when working on the apmpackage (see https://github.com/elastic/apm-server/pull/3464/files).